### DISCUSSION TITLE
MIB1 Canton with MIB2 unit

### DISCUSSION THREAD

**Discussion by MichalHarajda** [Mr-MIBonk/M.I.B._More-Incredible-Bash#518] [Q&A]
It looks like dataset for mib1 does not work. I try to test it today, but I cannot make old canton amplifier to work with MIB2 unit.  Before there was APG firmware and it works with old canton...
Any ideas ?

**Comment by MichalHarajda** [comment]
nobody ?

**Reply by michaelvn** [reply]
Hello there, I've got the same problem tried everything can't make MHI2 to work with old MIB1 Canton.
The amp doesn't even turn on so I think it's more than just 3B00 dataset.
You mentioned there's APG firmware, can you share?
Thanks!
